Source/Blazorise/wwwroot/datePicker.js

```diff
@@ -52,7 +52,7 @@ export function initialize(element, elementId, options) {
         clickOpens: !(options.readOnly || false),
         disable: options.disabledDates || [],
         inline: options.inline || false,
-        disableMobile: options.disableMobile || true,
+        disableMobile: options.disableMobile || false,
```

Comment:
maybe I'm dumb, what is this? options.disableMobile || false? is this some kind of way to coalesce?